Jetcaster/wear/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -80,10 +83,20 @@ private fun PlayerScreen(
     PlayerScreen(
         mediaDisplay = {
             if (playerUiState != null) {
-                playerUiState.episodePlayerState.currentEpisode?.let {
+                if (playerUiState.episodePlayerState.currentEpisode != null) {
```

Comment:
2 null checks are being done here. You can simplify this to:

```
val episode = playerUiState.episodePlayerState.currentEpisode
if (episode != null) {
     TextMediaDisplay(
         title = episode.podcastName,
         subtitle = episode.title
     )
} else {
    TextMediaDisplay(
          title = it.podcastName,
          subtitle = it.title
          title = "Nothing to play",
          subtitle = ""
      )
}
```

Reply:
In my check I am also checking that the tile is not "" as I want to show "Nothing to play" instead (data class PlayerEpisode sets the title to "")

Reply:
How about?

```
val episode = playerUiState.episodePlayerState.currentEpisode
if (episode != null && episode.title.isNotEmpty()) {
     TextMediaDisplay(
         title = episode.podcastName,
         subtitle = episode.title
     )
} else {
    TextMediaDisplay(
          title = it.podcastName,
          subtitle = it.title
          title = "Nothing to play",
          subtitle = ""
      )
}
```